### PR TITLE
Correcting calculation of confidence interval

### PR DESCRIPTION
**Pull request by paolzpk**
Hello @RonsenbergVI , I was trying to use your library to plot some confidence intervals and I wasn't getting satisfying results.
I don't have any way to check the formulas, but to use my rusty statistics knowledge. 
However, I think the standard deviation is wrongly calculated and the mean estimator term @74 should not go under the square root.
As far as I know T ~ (x - mean)/(s/sqrt(N)), hence inverting: t*s/sqrt(n) + mean.
With these changes, I get some better results on a dataset of mine. Let me know if you agree!

In summary:
- Correcting std dev calculation using numpy;
- Correcting Student T formula using T ~ (x - mean)/(s/sqrt(N)) --> t*s/sqrt(n) + mean.

### CODE REVIEW

**Comment by RonsenbergVI** [comment]
hey,
thanks for the change, I merged your branch with the current changes so I can see how it looks (I think there is a parenthesis too many in your commit) and come back to you with remarks (and hopefully merge your PR)
